src/d3a/d3a_core/simulation.py

```diff
@@ -380,7 +380,7 @@ def _handle_input(self, console, sleep: float = 0):
                     self.stop()
                 elif cmd == '+':
                     v = 5
-                    if self.slowdown <= 95:
+                    if self.slowdown <= 9995:
```

Comment:
Can you tell the reason for this change?

Reply:
The reason for this change is because the max slowdown is 10000, not 100. This had the effect that I could not slow down the runtime of the simulation enough. If you agree though, I can change the max slowdown to 100 globally, so that it can be associated with a ratio of delay (0-no delay, 100-realtime).